lib/utils/combineJSON.js

```diff
@@ -35,12 +36,13 @@ function combineJSON(arr, deep, collision) {
   }
 
   for (i = 0; i < files.length; i++) {
-    var file_content = fs.readFileSync(files[i], 'utf8');
+    var resolvedPath = resolveCwd(path.isAbsolute(files[i]) ? files[i] : '.' + path.sep + files[i]);
```

Comment:
Is this module needed? Is there an issue with relative paths in `require()` on line 43?

Reply:
Yeah, the node resole alg. is basically (check me on this...)

> if `./some/path.js` or `../some/path.js` start at the module you're in
> if `some/path.js` resolve a node_module of name `some` and then the file path `path.js`

There's no part of it really that's "start at cwd() and give me file some/path relative to that".

Reply:
Could leave that up to the end user to make process.cwd() part of the glob pattern, too, I suppose.

Reply:
Ah ok, I see now. It's a difference in how fs.readFile and require resolve the file paths, and because we are changing how we get the file contents from fs to require this is to ensue it is backwards compatible. Thank you for explanation and care to make sure this doesn't break backwards compatibility.

Reply:
the traditional method is to use fs.readFileSync(__dirname + './my-file')).  The __dirname global is baked into node.js - https://nodejs.org/docs/latest/api/modules.html#modules_dirname